src/main/java/com/google/devtools/build/lib/vfs/OsPathPolicy.java

```diff
@@ -86,14 +86,23 @@ public interface OsPathPolicy {
 
   boolean isCaseSensitive();
 
-  // We *should* use a case-insensitive policy for OS.DARWIN, but we currently don't handle this.
-  OsPathPolicy HOST_POLICY =
-      OS.getCurrent() == OS.WINDOWS ? WindowsOsPathPolicy.INSTANCE : UnixOsPathPolicy.INSTANCE;
+  OsPathPolicy HOST_POLICY = getFilePathOs(OS.getCurrent());
```

Comment:
nit: `SERVER_POLICY`? ("host" is ambiguous, because it's also the old name for "exec")

For this reason, a comment would also be welcome.

Reply:
My understanding (per https://bazel.build/extending/platforms) is that we've retroactively redefined "host" to mean "the platform Bazel runs on" and introduced "exec" to mean "the platform where an action executes". The only reason it's ambiguous is that we haven't followed through in replacing "host" with "exec" everywhere it matters (notably, `--host_*` flags which should really be called `--exec_*`).

A comment would be welcome, yes, but IMO we should stick to "host" or "exec" instead of introducing yet another term.